index.html

```diff
@@ -20,19 +20,21 @@
     <a href="logo" class="logo">My logo</a>
     <nav>
       <div class="mobile-navbar" id="mobile-navbar">
+        <div class="overlay">
+        <img class="exit-btn" width="20" src="./images/Exit-button.svg" alt="hamburger cancel button">
         <ul class="mobile-nav-menu" id="mobile-nav-menu">
-          <li class="mobile-nav-link"><a href="#works">Portfolio</a></li>
-          <li class="mobile-nav-link"><a href="#about-myself">About</a></li>
-          <li class="mobile-nav-link"><a href="#form-section">Contact</a></li>
+          <li class="mobile-nav-link"><a href="#works-sect">Portfolio</a></li>
+          <li class="mobile-nav-link"><a href="#about-me">About</a></li>
+          <li class="mobile-nav-link"><a href="#form">Contact</a></li>
         </ul>
+        </div>
         <div class="menu-button">
           <img class="menu-button" width="18" height="12" src="images/Union.svg" alt="menu-icon">
         </div>
       </div>
     </nav>
   </header>
-
-  <section class="headline" id="headline"> 
+  <div class="headline" id="headline"> 
```

Comment:
[ ] [optional] @sharon-odhiambo great work implementing the changes, kindly check on your design the background images are not in the desired position like in the Figma design:
![image](https://user-images.githubusercontent.com/61408860/174869585-1efc9af6-6ac0-48a5-adac-092864f4572c.png)